src/NetTopologySuite/IO/GML2/GMLReader.cs

```diff
@@ -161,14 +168,15 @@ protected Coordinate ReadPosAsCoordinate(string[] value)
         /// <summary>
         /// Extract a <see cref="Coordinate" /> from a x,y string value.
         /// </summary>
+        [Obsolete]
```

Comment:
Suggestion: I personally always try to add messages indicating what current callers should use instead.  If we're not **actually** expecting people to inherit from this class, then it might be a decent idea to add this message to all the `protected` members (though that might also be overkill):
```suggestion
        [Obsolete("This class should generally not be inherited from.  Protected members do not necessarily have a perfectly stable API.")]
```